R-packages/evalcast/R/get_predictions.R

```diff
@@ -69,22 +83,36 @@ get_predictions <- function(forecaster,
                             apply_corrections = function(signals) signals,
                             response_data_source = signals$data_source[1],
                             response_data_signal = signals$signal[1],
-                            forecaster_args = list()) {
-  
+                            parallel_execution = TRUE,
+                            forecaster_args = list(),
```

Comment:
For backwards compatibility with code calling args positionally, might want to swap the order of `parallel_execution` and `forecaster_args` here.